.github/ISSUE_TEMPLATE/20-docs-issue.md

```diff
@@ -0,0 +1,46 @@
+Thank you for submitting a TFX documentation issue. Per our GitHub
+policy, we only address code/doc bugs, performance issues, feature requests, and
+build/installation issues on GitHub.
+
+The TFX docs are open source! To get involved, read the documentation
```

Comment:
..., and we welcome external contributions!

Reply:
Done